libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.html

```diff
@@ -25,9 +26,17 @@
     </sky-data-manager>
   </sky-modal-content>
   <sky-modal-footer>
+    <sky-alert
```

Comment:
The screenshot in the spec shows a status indicator instead of an alert.

Reply:
resolved in new PR. closing this PR.